doc/start/documenting-ceph.rst

```diff
@@ -60,32 +60,33 @@ instead of compiling program source. This sequence (the sequence of building the
 Get the Source
 --------------
 
-Ceph documentation lives in the Ceph repository right alongside the Ceph source
-code under the ``ceph/doc`` directory. For details on github and Ceph,
-see :ref:`Get Involved`.
+The source of the Ceph documentation is a colletion of ReStructured Text files
+that are in the Ceph repository in the ``ceph/doc`` directory.  For details
+on github and Ceph, see :ref:`Get Involved`.
 
 The most common way to make contributions is to use the `Fork and Pull`_
```

Comment:
Is there any other??

Reply:
I've rewritten it so that it won't plant the seed of doubt in the mind of the curious neophyte.